shell/common/node_bindings.cc

```diff
@@ -151,6 +152,25 @@ void V8FatalErrorCallback(const char* location, const char* message) {
   *zero = 0;
 }
 
+bool AllowWasmCodeGenerationCallback(v8::Local<v8::Context> context,
+                                     v8::Local<v8::String>) {
+  // If we're running with contextIsolation enabled, fall back to Blink's logic.
+  // We switch on the embedder data field count here because if we're in a ctx
+  // that Node.js doesn't control, the number of embedder data fields will
+  // always be less than the min index Node.js uses.
+  if (context->GetNumberOfEmbedderDataFields() <
+      node::ContextEmbedderIndex::kEnvironment) {
+    v8::Isolate* isolate = context->GetIsolate();
+    LOG(INFO) << __LINE__;
+    return blink::V8Initializer::WasmCodeGenerationCheckCallbackInMainThread(
+        context, v8::String::Empty(isolate));
+  }
+
+  v8::Local<v8::Value> wasm_code_gen = context->GetEmbedderData(
```

Comment:
Can we phone into nodes logic here instead of copying it?  This method would ideally be:

```js
if (isNodeContext(context))
  return nodeThing();

return blinkThing();
```

Might have to export nodes thing too

Reply:
node might be more reticent to do that - can we add it as a todo item and refactor this if they're amenable so we don't pre-emptively add a patch we can't get rid of?

Reply:
Without referencing it exactly I'm not sure how we'd keep it in sync 🤔  I think the one-line patch we might have to keep around forever is OK (though I'd still like it upstreamed).  It's just a `NODE_EXPORT` right?